ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/models/Element.java

```diff
@@ -0,0 +1,140 @@
+package com.spectralogic.ds3autogen.c.models;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3autogen.api.models.Ds3Annotation;
+import com.spectralogic.ds3autogen.utils.Helper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.text.ParseException;
+
+
+public class Element {
+    private static final Logger LOG = LoggerFactory.getLogger(Element.class);
+
+    private final String name;
+    private final String type;
+    private final String componentType;
+    private final ImmutableList<Ds3Annotation> annotations;
+
+    public Element(
```

Comment:
This class seems to have too many responsibilities in it.  It looks like that in addition to it being a model, it also has knowledge about how it should be displayed in the generated code.  Split these two responsibilities out.